### ISSUE TITLE
Selectize dropdowns have low contrast

### ISSUE THREAD

**Issue by matthew-li** [ucb-rit/coldfront#269]
After the accessibility changes (#221), the options in the dropdowns using `selectize.js` have black text on a dark blue background when highlighted, which is very difficult to read.

The best option is to change this highlight to the light gray that it used to be. The color seems to have been affected by the change to `static.common.css.common.css` (likely the `.active` class?).

The second best option would be to disable `selectize` for these dropdowns, although that would remove the search feature, making them harder to use.
